dataprofiler/profilers/unstructured_text_profile.py

```diff
@@ -0,0 +1,223 @@
+from collections import defaultdict
+from . import utils, BaseColumnProfiler
+import itertools
+
+class TextProfiler(object):
+    type = 'unstructured_text'
+
+    def __init__(self, name, options=None):
```

Comment:
should this take in name? is it the name of the file or dataset? not sure how we are dealing with this since textdata

Reply:
should this even have name?

Reply:
i'd vote for removing the name

Reply:
After talking, we decided to leave it for now